FSharpLu.Json/Script.fsx

```diff
@@ -1,10 +1,10 @@
 ﻿#r @"System.dll"
 #r @"System.Core.dll"
-#r @"..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll"
+#r @"C:\Users\wiblum\.nuget\packages\Newtonsoft.Json\10.0.3\lib\net45\Newtonsoft.Json.dll"
```

Comment:
Good catch. I'm not sure what is the best way to address it though. With .Net core the nuget package directory is global and is located under the user profile instead of the source directory. I am thinking to just leave a placeholder in the .fsx file and let each developer adjust it on their local machine:
```
#r @"%USERPROFILE%\.nuget\packages\Newtonsoft.Json\10.0.3\lib\net45\Newtonsoft.Json.dll"
```